applications/OptimizationApplication/custom_utilities/container_properties_data_io.h

```diff
@@ -0,0 +1,58 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   license: OptimizationApplication/license.txt
+//
+//  Main author:     Suneth Warnakulasuriya
+//
+
+#pragma once
+
+// System includes
+
+// Project includes
+#include "containers/variable.h"
+#include "containers/container_expression/container_data_io.h"
+#include "includes/model_part.h"
+
+// Application includes
+
+namespace Kratos {
+
+///@name Kratos Classes
+///@{
+
+namespace ContainerDataIOTags {
+    struct Properties    {};
+} // namespace Tags
+
+template <>
+struct ContainerDataIO<ContainerDataIOTags::Properties>
+{
+    static constexpr std::string_view mInfo = "Properties";
```

Comment:
Props for using `std::string_view`!

Reply:
Thats the only one which is a `constexpr` for strings as well :D